Tests/SonarQube.Bootstrapper.Tests/ArgumentProcessorTests.cs

```diff
@@ -91,15 +141,15 @@ public void ArgProc_PropertyOverriding()
 
             // 1. Settings file only
             IBootstrapperSettings settings = CheckProcessingSucceeds(logger, "/s: " + fullPropertiesPath);
-            AssertExpectedUrl("http://settingsFile", settings);
+            AssertCommonSettings(settings, "http://settingsFile", true);
```

Comment:
It quickly became clear that always checking the childCmdLineArgs is very tedious because we have to escape quotes so I let a few cases slide because we already have coverage over this argument type.